cmake/modules/RootNewMacros.cmake

```diff
@@ -1079,6 +1079,22 @@ function(ROOT_ADD_TEST test)
 endfunction()
 
 #----------------------------------------------------------------------------
+# function ROOT_ADD_GTEST(<testsuite> <name> LIBRARIES)
+#
+function(ROOT_ADD_GTEST test_suite test_name)
```

Comment:
Yeah, that could be done when we start adding tests and need it. 

I don't like globbing because it causes quite a few issues if the source folder is unclean, etc. In general, it would be much more often editing tests rather than adding new ones.